### DISCUSSION TITLE
Welcome to Discussions! (Click here for a short introduction)

### DISCUSSION THREAD

**Discussion by TitusPosiadly** [MKT-Playbook/Hub#1] [Announcements]
## 👋 Welcome!
  Glad you made your way here. I'm using Discussions as a place to connect with the users of my app and other members of the community. We hope that you:
  * Ask questions you’re wondering about.
  * Share your ideas and suggestions.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, say "Hi 🙋" in the comment down below and tell us what's on your mind. Or have a look at the following pointers for the various topics.

### 📣 Announcements
All important messages and news can be found right here.

### 💬 General
Just want to chat a bit? We are excited to hear what you have to say.

### 💡 Feature requests
This is the right place to suggest new features or vote on existing ones. Feel free to join in on the discussion if you see a feature you like.

### 🗳 Polls
From time to time I will start some polls regarding new features and ideas to help me to get my priorities right. Feel free to post your own polls right here as well.

### 🙏 Q&A
Some parts of the app are unclear to you? Don't worry we're here to help! Ask to your heart's content and the community and I will answer as fast as we can.

### 🙌 Show and tell
Share your stories, screenshots, or videos right here. We would love to see them!

**Comment by jcapelle13** [comment]
Hi 🙋

I was updating for the animal tour and noticed that the gold rambi rider isn't in the db yet. Have I just not received the latest version yet? The App Store says I'm up to date

**Reply by TitusPosiadly** [reply]
Yeah, there seems to be an issue currently where sometimes not all the new data gets downloaded by the app. I will push another update to the public database in a sec. That should trigger your app to download everything once again. Let me know if it is still missing after about 10 minutes or so.

**Reply by jcapelle13** [reply]
That fixed it! Thank you!

**Reply by TitusPosiadly** [reply]
No problem. Glad I could help
